src/System.Private.Xml/src/System/Xml/Xsl/Xslt/XPathPatternParser.cs

```diff
@@ -164,7 +164,7 @@ private QilNode ParseRelativePathPattern()
         {
             if (++_parseRelativePath > MaxParseRelativePathDepth)
             {
-                if (LocalAppContextSwitches.LimitXPathComplexity)
```

Comment:
>LocalAppContextSwitches [](start = 20, length = 23)

I didn't realize we already have code that uses LocalAppContextSwitches. 

I think it would be better to keep using that name (LocalAppContext) to reduce the churn.